trame_vtk/tools/vtksz2html.py

```diff
@@ -0,0 +1,60 @@
+import argparse
+import base64
+from pathlib import Path
+import sys
+
+HTML_VIEWER_PATH = Path(__file__).with_name("static_viewer.html")
+
+
+def embbed_data_to_viewer(input_data_file_path):
```

Comment:
I'm wondering if we could set up the `export()` method on the widget view to return a memory buffer that could be directly passed to this to prevent intermediate file IO?

Reply:
Sure that is a possibility. You see how it is used.

Reply:
Merged it for now. We can revisit those helper methods and handle that buffer